compass/Coordinator.java

```diff
@@ -297,9 +297,10 @@ private void start() throws InterruptedException {
       }
 
       // If all the above checks pass we are ready to issue a new milestone
-      state.latestMilestoneIndex++;
-
       createAndBroadcastMilestone(trunk, branch);
+
+      // We increase the latest Milestone Index
+      state.latestMilestoneIndex++;
```

Comment:
this changes the logic in a few fundamental ways:
1. will reissue (sign a new version) the same milestone index in the state file on start. 
2. it will try rebroadcast the milestone b/c it's waiting for `latestMilestoneIndex` to match the node's latestMilestoneIndex, but we never issued such a milestone.

the way I see it it's a UI thing that the flag `index` should be last issued milestone and not the next milestone.
we can change the input parsing (add a -1) but we should not touch the main logic